<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Default test"
           tests="3"
           failures="1"
           errors="0"
           skipped="2"
           time="0.300">
    <properties></properties>

            <testcase
                name="config"
                time="0.300"
                classname="Dealership.Masterdata">

            <failure type="java.lang.NoSuchFieldError" message="VERSION_2_3_30">
                <![CDATA[java.lang.NoSuchFieldError: VERSION_2_3_30
                com.aventstack.extentreports.reporter.AbstractFileReporter.loadTemplateModel(AbstractFileReporter.java:54)com.aventstack.extentreports.reporter.ExtentSparkReporter.start(ExtentSparkReporter.java:136)com.aventstack.extentreports.reporter.ExtentSparkReporter.access$000(ExtentSparkReporter.java:27)com.aventstack.extentreports.reporter.ExtentSparkReporter$1.onSubscribe(ExtentSparkReporter.java:116)io.reactivex.rxjava3.subjects.PublishSubject.subscribeActual(PublishSubject.java:133)io.reactivex.rxjava3.core.Observable.subscribe(Observable.java:13099)com.aventstack.extentreports.ReactiveSubject.attachReporter(ReactiveSubject.java:38)com.aventstack.extentreports.ExtentReports.attachReporter(ExtentReports.java:66)Dealership.Masterdata.config(Masterdata.java:42)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)java.base/java.lang.reflect.Method.invoke(Method.java:566)org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:62)org.testng.internal.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:385)org.testng.internal.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:321)org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:637)org.testng.TestRunner.beforeRun(TestRunner.java:627)org.testng.TestRunner.run(TestRunner.java:589)org.testng.SuiteRunner.runTest(SuiteRunner.java:377)org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)org.testng.SuiteRunner.run(SuiteRunner.java:276)org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)org.testng.TestNG.runSuitesLocally(TestNG.java:1134)org.testng.TestNG.runSuites(TestNG.java:1063)org.testng.TestNG.run(TestNG.java:1031)org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </failure>
    </testcase>

            <testcase
                name="AddCustomer"
                time="0.0"
                classname="Dealership.Masterdata">
            </testcase>
            <testcase
                name="Test2"
                time="0.0"
                classname="Dealership.Masterdata">
            </testcase>

</testsuite>
